llrt_core/src/vm.rs

```diff
@@ -184,6 +184,28 @@ impl Vm {
 
         self.run(source, strict, global).await;
     }
+
+    pub async fn run_repl(&self) {
+        self.ctx
+            .with(|ctx| -> Result<()> {
+                let global = ctx.globals();
+                let console: Object = global.get("console")?;
+                let js_log: Function = console.get("log")?;
+                loop {
+                    let mut input = String::new();
```

Comment:
We don't need to create a new string very time. We can create one outside the loop and clear it.